python/lsst/pipe/tasks/matchFakes.py

```diff
@@ -148,6 +178,39 @@ def run(self, fakeCat, diffIm, associatedDiaSources):
                 associatedDiaSources.reset_index(drop=True), on="diaSourceId", how="left")
         )
 
+    def composeFakeCat(self, fakeCats, skyMap):
+        """Concatenate the fakeCats from tracts that may cover the exposure.
+
+        Parameters
+        ----------
+        fakeCats : `list` of `lst.daf.butler.DeferredDatasetHandle`
+            Set of fake cats to concatenate.
+        skyMap : `lsst.skymap.SkyMap`
+            SkyMap defining the geometry of the tracts and patches.
+
+        Returns
+        -------
+        combinedFakeCat : `pandas.DataFrame`
+            All fakes that cover the inner polygon of the tracts in this
+            quantum.
+        """
+        if len(fakeCats) == 1:
+            return fakeCats[0].get(
+                datasetType=self.config.connections.fakeCats)
+        outputCat = []
+        for fakeCatRef in fakeCats:
+            cat = fakeCatRef.get(
+                datasetType=self.config.connections.fakeCats)
+            tractId = fakeCatRef.dataId["tract"]
+            # Make sure all data is within the inner part of the tract.
+            outputCat.append(cat[
+                skyMap.findTractIdArray(cat[self.config.raColName],
+                                        cat[self.config.decColName],
```

Comment:
@morriscb shouldn't these column names be `self.config.ra_col` and `self.config.dec_col` given that `self.config.raColName` and `self.config.decColName` are going to be deprecated?